### PR TITLE
chore(content-picker): Added stories and mdx

### PR DESCRIPTION
**Pull request by greg-in-a-box**
image

migrated the old readme to this mdx file 

image

I dont know why ContentExplorer is after ContentPicker since its supposed to sorted, maybe its because its a folder

### CODE REVIEW

File: src/elements/content-picker/stories/ContentPicker.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { Meta } from '@storybook/blocks';
+
+import ContentPickerStories from './ContentPicker.stories';
+
+<Meta of={ContentPickerStories} />
```
Comment (greg-in-a-box) [inline src/elements/content-picker/stories/ContentPicker.mdx:5]: Needed this so that it places the doc in the right location

File: package.json [hunk 2]
```diff
@@ -293,6 +294,7 @@
         "react-tether": "1.0.5",
         "react-textarea-autosize": "^8.5.3",
         "regenerator-runtime": "^0.13.3",
+        "remark-gfm": "^4.0.0",
```
Comment (greg-in-a-box) [inline package.json:297]: Needed to install this to get the tables to render correctly 
https://storybook.js.org/docs/writing-docs/mdx#markdown-tables-arent-rendering-correctly

File: src/elements/content-picker/stories/ContentPicker.mdx (hunk 1 above)
Comment (jpan-box) [reply src/elements/content-picker/stories/ContentPicker.mdx:5]: thanks for pointing this out! I went and read about how this works here:
https://storybook.js.org/docs/api/doc-block-meta

File: src/elements/content-picker/stories/ContentPicker.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+---
+
+
+### Props
+
+
+| Prop                           | Type                      | Default          | Description                                                                                                                                                   |
+|--------------------------------|---------------------------|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| token*                         | string                    |                  | *See the [developer docs](https://developer.box.com/guides/embed/ui-elements/picker/#Options).*                                                               |
```
Comment (jpan-box) [inline src/elements/content-picker/stories/ContentPicker.mdx:45]: Do you like using developer.box.com links? Or what about github links? i have no preference either way, just wanted to see what you thought

**Review by jpan-box — APPROVED** [review]

File: src/elements/content-picker/stories/ContentPicker.mdx (hunk 3 above)
Comment (greg-in-a-box) [reply src/elements/content-picker/stories/ContentPicker.mdx:45]: the context of the MDX is from the original readmes, I would use them for now till we can use typescript to auto generate stuff

**Review by QingyuChai — APPROVED** [review]

**Pull request merged**